wandb/sdk/lib/wburls.py

```diff
@@ -0,0 +1,42 @@
+"""Container for urls used in the wandb package.
+
+Use this anytime a URL is displayed to the user.
+
+Usage:
+    ```python
+    from wandb.sdk.lib.wburls import wburls
+
+    print(f"This is a url {wburls.get('cli_launch')}")
+    ```
+"""
+
+from typing import Dict, Optional
+from typing import TYPE_CHECKING
+
+
+if TYPE_CHECKING:
+    from ._wburls_generated import URLS
+
+
+class WBUrls:
+    _urls_dict: Optional[Dict["URLS", str]]
+
+    def __init__(self) -> None:
+        self._urls_dict = None
+
+    def _get_urls(self) -> Dict["URLS", str]:
+        return dict(
+            cli_launch="http://wandb.me/launch",
+            doc_run="https://wandb.me/run",
+            doc_require="http://wandb.me/library-require",
+            doc_start_err="https://docs.wandb.ai/library/init#init-start-error",
+            upgrade_local="https://wandb.me/local-upgrade",
+        )
+
+    def get(self, s: "URLS") -> str:
```

Comment:
nit: it would be nice to use `__getitem__` for convince, what do you think?

Reply:
i think this is fine.. more explicit the better also it forces you not to specify a default, i want it to assert on not found